repository-data/webfiles/src/main/resources/site/freemarker/include/debugger.ftl

```diff
@@ -91,7 +90,7 @@
 
             <#if debugObject?is_number && debugObject == -1>
                 <div class="${settings.styleClassPrefix}-error">
```

Comment:
Is this required for this ticket? , removing xhtml

Reply:
debugger.ftl should not have been committed, so it has now been reverted